src/System.IO.Abstractions.TestingHelpers/MockFileInfo.cs

```diff
@@ -167,6 +167,18 @@ public override DateTime LastWriteTimeUtc
             }
         }
 
+#if FEATURE_FILE_SYSTEM_INFO_LINK_TARGET
+        /// <inheritdoc />
+        public override string LinkTarget
+        {
+            get
+            {
+                var mockFileData = GetMockFileDataForWrite();
```

Comment:
I guess this just needs to read:

```suggestion
                var mockFileData = GetMockFileDataForRead();
```

Reply:
:man_facepalming: